performance-tests/bench_2/idl/Bench.idl

```diff
@@ -35,6 +35,7 @@ module Bench {
     Builder::PropertySeq params;
     Builder::StringSeq readers;
     Builder::StringSeq writers;
+    string content_filtered_topic;
```

Comment:
Let's not add this here, I wouldn't expect most actions to know about content filtered topics.